### DISCUSSION TITLE
New Build from Master?

### DISCUSSION THREAD

**Discussion by kentdlee** [capnproto/capnproto#2001] [General]
Kenton,

Can we get a new build from master on github. It looks like pycapnp pulls from the latest build on github and we are trying to get pycapnp updated to get rid of the unwanted warning messages. I believe this is what we need for this to happen, but if there is something I am missing please feel free to tell me what I should be doing to get this taken care of. 

Thanks!

  -- Kent

**Comment by kentonv** [comment]
Sorry, I'm not sure what you mean here. Do you mean you want me to roll a new 1.x release? I don't publish releases via github (probably should but never have). Releases are published on the web site only. What is pycapnp looking for exactly? Maybe it looks for the latest release tag?

Anyway if the request is to do a release, I'll put it on my todo list, but can't promise when I'll get to it. But I'd expect there's some way to build pycapnp against any random commit of capnproto that you want, or against an installed version of the library...
